src/VPolytope.jl

```diff
@@ -250,6 +249,34 @@ function cartesian_product(P1::VPolytope{N}, P2::VPolytope{N};
     return VPolytope(Pcp)
 end
 
+"""
+    remove_redundant_vertices(P::VPolytope{N};
+                              backend=default_polyhedra_backend(P, N))::VPolytope{N} where {N}
+
+Return the polytope obtained by taking the convex hull of the vertices of the
+given polytope.
```

Comment:
I find this summary confusing. The function should remove redundant vertices. (same for `VPolygon`)

Reply:
changing to: "Return the polytope obtained by removing the redundant vertices of the given polytope".